bika/lims/browser/instrument.py

```diff
@@ -538,39 +503,38 @@ def __init__(self, context, request, **kwargs):
         BikaListingView.__init__(self, context, request, **kwargs)
         self.form_id = "instrumentcertifications"
         self.columns = {
-            'Title': {'title': _('Cert. Num'),
-                      'index': 'sortable_title'},
-            'getAgency': {'title': _('Agency')},
-            'getDate': {'title': _('Date')},
-            'getValidFrom': {'title': _('Valid from')},
-            'getValidTo': {'title': _('Valid to')},
-            'getDocument': {'title': _('Document')},
+            'Title': {'title': _('Cert. Num'), 'index': 'sortable_title'},
+            'getAgency': {'title': _('Agency'), 'sortable': False},
+            'getDate': {'title': _('Date'), 'sortable': False},
+            'getValidFrom': {'title': _('Valid from'), 'sortable': False},
+            'getValidTo': {'title': _('Valid to'), 'sortable': False},
+            'getDocument': {'title': _('Document'), 'sortable': False},
         }
         self.review_states = [
             {'id':'default',
              'title':_('All'),
              'contentFilter':{},
-             'columns': [ 'Title',
+             'columns': ['Title',
                          'getAgency',
                          'getDate',
                          'getValidFrom',
                          'getValidTo',
                          'getDocument'],
-             'transitions': [{}]},
+             'transitions': []},
         ]
         self.allow_edit = False
         self.show_select_column = False
-        self.show_workflow_action_buttons = False
+        self.show_workflow_action_buttons = True
```

Comment:
Afak, there are no specific transitions available for Certifications (or at least, shouldn't be). Why the need of displaying the workflow action buttons bar?

Reply:
Indeed, good catch @xispa . I had to add this to display the bottom button bar where the "Show more" button lives.

Reply:
I see, it was just a required trick